### ISSUE TITLE
Autoupgrade - 1.7.8.8 to 8.0.3 fails 

### ISSUE THREAD

**Issue by djoelleuch** [PrestaShop/PrestaShop#31978]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

i tried to upgrade PS 1.7.8.8 to PS 8.0.3 with PHP 7.4 / PHP 7.3 , but it fails with 26 errors 

![image](https://user-images.githubusercontent.com/115152460/228851147-7c381f54-f032-4915-91da-18dbdfc5be46.png)

### Expected behavior

A  success upgrade .

### Steps to reproduce

1. Go to BO > Advanced Parameters > Performance
2. Disable the cache
3. Enable Debug Mode
4. Install Autoupgrade
5. Clic on Configure
6. Set up your local archive to have the 8.0.3 on it
7. Select on the expert mode your local archive and your different zip/xml
8. Save it
9. Launch the Autoupgrade
10. See ERRORS ! 

### PrestaShop version(s) where the bug happened

1.7.8.8

### PHP version(s) where the bug happened

7.4 / 7.3 

### If your bug is related to a module, specify its name and its version

Autoupgrade v4.15

**Comment by djoelleuch** [comment]
it is maybe  related to this[ issue](https://github.com/PrestaShop/PrestaShop/issues/31679) but it's not the same errors .

**Comment by florine2623** [comment]
Hello @djoelleuch ,

I couldn't reproduce your issue 😓 

I upgraded from 1.7.8.8 to 8.0.3. The upgrade was successful :
<img width="1425" alt="Screenshot 2023-03-30 at 17 56 42" src="https://user-images.githubusercontent.com/16019289/228895217-4fb9b04b-585f-43b0-a400-1995267c43d2.png">


https://user-images.githubusercontent.com/16019289/228895251-7a067492-8b8b-4acc-9176-e29fcaa2b6cb.mov

I only have some elements that are not displayed correctly in the BO

Maybe @PrestaShop/qa-functional can retest this case ?

**Comment by jolelievre** [comment]
Hi @djoelleuch 

can you give more details about the error? Especially we'd need the full log of the error, on your screenshot it is cut. Besides the numbers you indicated are not the number of errors, it's actually the stack trace of a single error that details the previous calls, so it indicates the 26 steps before the code failed.

**Comment by HanaRebaiQA** [comment]
Hello @djoelleuch 

Thank you for your issue.
I was able to reproduce your issue only when not using the XML. But, I was unable to reproduce it when using it.

With XML : OK :heavy_check_mark: 
![image](https://user-images.githubusercontent.com/92912932/229141782-b30411ab-33a3-4186-b081-1b2e41721cc8.png)

Without XML : NOK :x: 

![image](https://user-images.githubusercontent.com/92912932/229141735-5d64b29b-3605-4466-9744-5411e5789712.png)

This issue has already been fixed by this [PR](https://github.com/PrestaShop/autoupgrade/pull/518) on the autoupgrade module v4.16

So, it's a duplicate of this issue #29932
